src/components/nestedList.js

```diff
@@ -0,0 +1,114 @@
+import { useState, useEffect, Fragment } from "react";
+import List from "@mui/material/List";
+import ListItemButton from "@mui/material/ListItemButton";
+import ListItemText from "@mui/material/ListItemText";
+import ListItem from "@mui/material/ListItem";
+import Collapse from "@mui/material/Collapse";
+import ExpandLess from "@mui/icons-material/ExpandLess";
+import ExpandMore from "@mui/icons-material/ExpandMore";
+
+/**
+ * Subcomponent that can created nested lists to display in DialogPopup
+ * @returns {JSX.Element}
+ */
+function NestedList() {
```

Comment:
be consistent with component declarations,
```suggestion
const NestedList = () => {
```